internal/server/rest.go

```diff
@@ -423,6 +426,35 @@ func pingHandler(w http.ResponseWriter, _ *http.Request) {
 	w.WriteHeader(http.StatusOK)
 }
 
+func sourceInfosManageHandler(w http.ResponseWriter, r *http.Request, st ast.StreamType) {
+	defer r.Body.Close()
+	var (
+		content []xsql.StreamInfo
+		err     error
+		kind    string
+	)
+	if st == ast.TypeTable {
+		kind = r.URL.Query().Get("kind")
+		if kind == "scan" {
+			kind = ast.StreamKindScan
+		} else if kind == "lookup" {
+			kind = ast.StreamKindLookup
+		} else {
+			kind = ""
+		}
+	}
+	if kind != "" {
+		content, err = streamProcessor.ShowTableInfo(kind)
+	} else {
+		content, err = streamProcessor.ShowStreamInfo(st)
+	}
```

Comment:
I think the users who upgraded eKuiper from old version can not use this api as here we read data from the database.

I think we can use `streamProcessor.DescStream` here to get the info we want.

Reply:
Get. I will revert the change the db schema. But I think DescStream might still be avoided by adding a new structure to return stream detail, I will change that later.

Reply:
Sorry. My thought was wrong. If we don't change the db schema, we have to descstream. And the code is updated.
But I'm wondering if some times in the future you have to change the database schema, how to deal with that?
I'm not quite familiar with SQLite, but I thought if we design the structure carefully, maybe it will always be forward compatible.

Like, if we only append new fields to the xsql.StreamInfo{}, the old data can still be serializable? I don't know if it is possible. Some database can do that automatically I guess.